doc/source/tune.rst

```diff
@@ -11,9 +11,20 @@ This document describes Ray.tune, a hyperparameter tuning tool for long-running
 
 -  Resource-aware scheduling, including support for concurrent runs of algorithms that may themselves be parallel and distributed.
 
+You can find the code for Ray.tune `here on GitHub <https://github.com/ray-project/ray/tree/master/python/ray/tune>`__.
+
+
+Installation
+------------
+
+Ray.tune has an extra YAML dependency on top of **ray**:
```

Comment:
pyyaml is pretty tiny, can we include it with Ray itself?

Reply:
Btw ray.tune's yaml dependency is soft; you can still use it (just without pretty printing, and you can't run yaml files).

You can probably just omit this section in that case.